docs/pages/eas-build-custom-builds/custom-build-config.mdx

```diff
@@ -103,9 +106,28 @@ Let's create a new [build profile](/build/eas-json/#build-profiles) called `test
   "build": {
     /* @hide ... */ /* @end */
     "test": {
+      "config": "test.yml",
+      "withoutCredentials": true
+    },
+}
+```
+
+We can also use the `withoutCredentials` option to skip the credentials setup on the CLI side as they are not needed for running tests.
+
+If you wish to use different, separate configs for each platform, you can create separate YAML config files for iOS and Android. For example:
```

Comment:
```suggestion
If you wish to use separate configs for each platform, you can create separate YAML config files for Android and iOS. For example:
```

Phrased as per [Referencing platforms](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web).